mne_features/univariate.py

```diff
@@ -29,7 +31,10 @@ def get_univariate_funcs(sfreq):
     Returns
     -------
     univariate_funcs : dict
+        The dictionary of feature functions.
     """
+    # XXX : you should make this automatically by inspecting the available
+    # functions that start with compute_
```

Comment:
for this @jbschiratti you could use something along these lines:

```
def get_univariate_funcs(sfreq):
    univariate_funcs = dict()
    for var in dir(univariate):
        if var.startswith("compute_"):
            var_short = var[len("compute_"):]
            univariate_funcs[var_short] = getattr(univariate, var)

    return univariate_funcs
```

that I would put in the `mne_features/__init__.py` file

Reply:
I totally agree! Now that the aliases of feature functions are clearly defined, `univariate_funcs` and `bivariate_funcs` dicts should be created automatically.